bccsp/types/idemixopts.go

```diff
@@ -350,6 +350,19 @@ type IdemixNymSignerOpts struct {
 	IssuerPK Key
 	// H is the hash function to be used
 	H crypto.Hash
+
+	// IsSmartcard checks whether this nym signature was generated by a smartcard
+	IsSmartcard bool
+
+	// Smartcard is a software smartcard used to support signing in s/w
+	Smartcard interface{}
```

Comment:
is interface{} the best we can have? It might cause issues with serialisation

Reply:
perfect 👍

Reply:
I added a comment though